notebooks/community/vizier/conversions_vertex_vizier_and_open_source_vizier.ipynb

```diff
@@ -0,0 +1,624 @@
+        "\n",
+        "If you are an existing OSS Vizier, the instruction shows you how to migrate the code to Vertex Vizier SDK. If you are an user of Vertex Vizier, the instruction shows how to modify the code to adapt OSS Vizier."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "AksIKBzZ-nre"
+      },
+      "source": [
+        "### Objective\n",
+        "\n",
+        "In this tutorial, you learn how to use `Vertex AI Vizier` to optimize a multi-objective study and convert the code to OSS Vizier.\n",
+        "\n",
+        "The goal is to __`minimize`__ the objective metric:\n",
+        "   ```\n",
+        "   y1 = r*sin(theta)\n",
+        "   ```\n",
+        "\n",
+        "and simultaneously __`maximize`__ the objective metric:\n",
+        "   ```\n",
+        "   y2 = r*cos(theta)\n",
+        "   ```\n",
+        "\n",
+        "that you will evaluate over the parameter space:\n",
+        "\n",
+        "   - __`r`__ in [0,1],\n",
+        "\n",
+        "   - __`theta`__ in [0, pi/2]\n",
+        "\n",
+        "### Costs\n",
+        "\n",
+        "This tutorial uses billable components of Google Cloud:\n",
+        "\n",
+        "* Vertex AI\n",
+        "\n",
+        "Learn about [Vertex AI\n",
+        "pricing](https://cloud.google.com/vertex-ai/pricing) and use the [Pricing\n",
+        "Calculator](https://cloud.google.com/products/calculator/)\n",
+        "to generate a cost estimate based on your projected usage.\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "iMHz63rPbq6P"
+      },
+      "source": [
+        "## Installation\n",
+        "\n",
+        "Install the packages required for executing this notebook."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "b6f3dc43494b"
+      },
+      "outputs": [],
+      "source": [
+        "import os\n",
+        "\n",
+        "# The Vertex AI Workbench Notebook product has specific requirements\n",
+        "IS_WORKBENCH_NOTEBOOK = os.getenv(\"DL_ANACONDA_HOME\") and not os.getenv(\"VIRTUAL_ENV\")\n",
+        "IS_USER_MANAGED_WORKBENCH_NOTEBOOK = os.path.exists(\n",
+        "    \"/opt/deeplearning/metadata/env_version\"\n",
+        ")\n",
+        "\n",
+        "# Vertex AI Notebook requires dependencies to be installed with '--user'\n",
+        "USER_FLAG = \"\"\n",
+        "if IS_WORKBENCH_NOTEBOOK:\n",
+        "    USER_FLAG = \"--user\"\n",
+        "\n",
+        "! pip3 install google-vizier==0.0.4\n",
```

Comment:
Do we also need to run the proto compilation script or not?

Reply:
No, since the proto compilation requires 3.8+ and colab only support 3.7, we are not going to test the OSS Vizier in the colab.